docs/topics/media-pipeline.rst

```diff
@@ -198,23 +198,25 @@ For self-hosting you also might feel the need not to use SSL and not to verify S
     AWS_VERIFY = False # or True (None by default)
 
 
-Custom Amazon S3 headers in FileStorage
+Custom Amazon S3 headers
+'''''''''''''''''''''''''
 
-Steps to pass **SSEKMSKeyId** and **ServerSideEncryption** to **scrapy** FilesPipeline using **AWS S3**
+Steps to pass **Headers** to **scrapy** FilesPipeline using **AWS S3**
 
-Configuration Steps::
+Configuration Steps:
 
-    1. Subclass ``scrapy.pipelines.files.S3FilesStore``, 
+| 1. Subclass :class:`~scrapy.pipelines.files.S3FilesStore`, 
```

Comment:
It’s the first time I see `|` at the beginning of a line in Sphinx, save for tables. What’s its purpose here?

Reply:
I wanted each step to be on a separate line, but now I've combined them into a paragraph, so I've removed these.

Reply:
I think it was actually better to keep them as separate, numbered steps.

The way to do that in reStructuredText is `#.  …`, not `| 1. …`. (you can use numbers instead of #, but # handles numbering automatically)